cohort_4/playbook/ACTIVITIES.md

```diff
@@ -1557,18 +1557,19 @@ An explanation of what this field is in day 1's feedback form (linked below).
 All of the feedback form responses will be put into the appropriate sheet in
 https://docs.google.com/spreadsheets/d/1D8y_UB11C4QrXB9kn86-UqVQQqT9FUDqftx1AMHIb3o.
 
-| Day | Link                                                                                         |
-| --- | -------------------------------------------------------------------------------------------- |
-|   1 | https://docs.google.com/a/zachlatta.com/forms/d/1FTBlvJdXRyfPyGheR3Sm6xWu1PQSMTNkZxPkC3jp0G4 |
-|   2 | https://docs.google.com/a/zachlatta.com/forms/d/1_9K3b4Nwgn3-mFqMrrUObwiMHt_53y9LVNyCQAULn7I |
-|   3 | https://docs.google.com/a/zachlatta.com/forms/d/1kgCrsPuTFJ-404JdpFo3acomXmwvTIOJHXzOp_NoUeI |
-|   4 | https://docs.google.com/a/zachlatta.com/forms/d/1ASfJdmXGUVRdGJR6rmzedDQ2EDEL7Hc11JD0xjBJMp8 |
-|   5 | https://docs.google.com/a/zachlatta.com/forms/d/1QL1IM9xi2GcU71Rr9dihDpJItRAsl9aA2mkH7jM2lm8 |
-|   6 | https://docs.google.com/a/zachlatta.com/forms/d/1y7WLP55aigI7Kf90wiy9z41xnqem-ltmT0myXJINzH0 |
-|   7 | https://docs.google.com/a/zachlatta.com/forms/d/15xAMDQmt4kCTCnbe9nZMqv5XUCIsxYaRMGEgLQNTNXI |
-|   8 | https://docs.google.com/a/zachlatta.com/forms/d/1bIkPi1IWIf34WXMHxivQDOqCWBIOq2SVUl_OFw2e_9c |
-|   9 | https://docs.google.com/a/zachlatta.com/forms/d/1LFVzviI9GDuxLTpOPWy9jNmImhmW0NisAMC0ocBJzIg |
-|  10 | https://docs.google.com/a/zachlatta.com/forms/d/1SAcCBYcr8qpzXjCWLlbeeilZUObBROR-EZh0WRTguLE |
+| Day  | Link                                                                                         |
+| ---- | -------------------------------------------------------------------------------------------- |
+|   1  | https://docs.google.com/a/zachlatta.com/forms/d/1FTBlvJdXRyfPyGheR3Sm6xWu1PQSMTNkZxPkC3jp0G4 |
+|   2  | https://docs.google.com/a/zachlatta.com/forms/d/1_9K3b4Nwgn3-mFqMrrUObwiMHt_53y9LVNyCQAULn7I |
+|   3  | https://docs.google.com/a/zachlatta.com/forms/d/1kgCrsPuTFJ-404JdpFo3acomXmwvTIOJHXzOp_NoUeI |
+|   4  | https://docs.google.com/a/zachlatta.com/forms/d/1ASfJdmXGUVRdGJR6rmzedDQ2EDEL7Hc11JD0xjBJMp8 |
+|   5  | https://docs.google.com/a/zachlatta.com/forms/d/1QL1IM9xi2GcU71Rr9dihDpJItRAsl9aA2mkH7jM2lm8 |
+|   6  | https://docs.google.com/a/zachlatta.com/forms/d/1y7WLP55aigI7Kf90wiy9z41xnqem-ltmT0myXJINzH0 |
+|   7  | https://docs.google.com/a/zachlatta.com/forms/d/15xAMDQmt4kCTCnbe9nZMqv5XUCIsxYaRMGEgLQNTNXI |
+|   8  | https://docs.google.com/a/zachlatta.com/forms/d/1bIkPi1IWIf34WXMHxivQDOqCWBIOq2SVUl_OFw2e_9c |
+|   9  | https://docs.google.com/a/zachlatta.com/forms/d/1LFVzviI9GDuxLTpOPWy9jNmImhmW0NisAMC0ocBJzIg |
+|  10  | https://docs.google.com/a/zachlatta.com/forms/d/1SAcCBYcr8qpzXjCWLlbeeilZUObBROR-EZh0WRTguLE |
+|  10+ | https://docs.google.com/a/hackmatch.com/forms/d/1C04DICTvylS_sDutVkzwe-e-MgxzABp9PGC0XPZVCkA |
```

Comment:
Can you add a description somewhere to this section describing what `10+` means?